### ISSUE TITLE
Idea for name

### ISSUE THREAD

**Issue by JamesGaier** [Nicholas-Baron/clicker_game#12]
What if we called it medieval kingdom simulator?

**Comment by Nicholas-Baron** [comment]
The most straightforward name for such a game.

**Comment by tapatiohaxx** [comment]
I agree that is very straightforward. It'll do for a name until we think of a more creative one, if possible.

**Comment by JamesGaier** [comment]
If anyone wants to post a name for the game fell free to add it here.  I am going to leave this issue open for that purpose.
